src/olympia/lib/crypto/tasks.py

```diff
@@ -22,9 +23,11 @@
 MAIL_MESSAGE = u"""
 Your add-on, {addon}, has been automatically signed for distribution in
 upcoming versions of Firefox. The signing process involved repackaging the
-add-on files and adding the string '.1-signed' to their versions numbers. The
-new versions have kept their review status and are now available for your
-users.
+add-on files and adding the string '.1-signed' to their versions numbers. If
+the version number already ended in '.1-signed', it was changed to
+'.1-signed-2'. The new versions have kept their review status and are now
+available for your users.
```

Comment:
Maybe something like "we increment the identifier for any resigning attempts, e.g '.1-signed-2'" but I trust a native english person more than me here :laughing: